net/lagrange/Portfile

```diff
@@ -20,26 +16,49 @@ maintainers         {@sikmir disroot.org:sikmir} openmaintainer
 description         A Beautiful Gemini Client
 long_description    {*}${description}
 
-checksums           rmd160  f7ad358149734f07d7009e31d47c3b78337765a5 \
-                    sha256  a9c3175acdc7bfd228cb4d24d68c4cfabe9a4fd79f821d631d556e8434749548 \
-                    size    9869222
+homepage            https://git.skyjake.fi/gemini/lagrange
+master_sites        ${homepage}/archive \
```

Comment:
what's the reason to move away from GitHub? The PG sets many thing automatically - for example, does the `livecheck` still work after this change?

Reply:
git.skyjake.fi is a primary hosting for `lagrange` repo and deps (`the_Foundation` and `sealcurses`), and only lagrange repo is mirrored on GitHub (`the_Foundation` and `sealcurses` are mirrored on codeberg.org). So just for consistency I think It's good to use git.skyjake.fi for all 3 ports. Moreover I've just found that we have PG `gitea 1.0`, so there is no problem with `livecheck`.